playbooks/gce/openshift-cluster/vars.yml

```diff
@@ -16,3 +16,98 @@ deployment_vars:
   enterprise: "{{ deployment_rhel7_ent_base }}"
   openshift-enterprise: "{{ deployment_rhel7_ent_base }}"
   atomic-enterprise: "{{ deployment_rhel7_ent_base }}"
+
+clusterid: mycluster
+region: us-central-1a
+
+provision:
+  clusterid: "{{ clusterid }}"
+  region: "{{ region }}"
+
+  build:  # build specific variables here
+    ami_name: "openshift-gi-"
+    base_image: ami-bdd5d6ab  # base image for AMI to build from
+    yum_repositories:  # this is an example repository but it requires sslclient info
```

Comment:
I moved these in a latest PR.  Feel free to do whatever you like here.  We can move or clean up later.